tests/cypress/support/step_definitions/formInputs.js

```diff
@@ -18,23 +18,33 @@ When("these form elements are edited:/filled:", function (dataTable) {
     const inputValue = row[2];
     switch (inputType) {
       case "singleCheckbox":
-        if (inputValue == "true") input.check().blur();
-        else input.uncheck();
+        if (inputValue == "true") {
+          input.check();
+          input.blur();
+        } else input.uncheck();
```

Comment:
Separating out chaining commands. You can read why from the cypress docs here: https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#Chains-of-Commands